src/hotspot/share/cds/aotCodeSource.cpp

```diff
@@ -0,0 +1,985 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If the code has been moved from other files, the current opinion/consensus is that the first copyright year should be the oldest first year from all the files from which the code was obtained.

Reply:
Since most of the code and logic are from filemap.[c|h]pp, I've updated the copyright year to 2003, 2025.